src/lib/github.js

```diff
@@ -81,32 +15,23 @@ module.exports = async function (pkg, info) {
     return;
   }
 
-  const githubUserFromGitConfig = parse.sync({path: gitConfigPath}).github
-    ? parse.sync({path: gitConfigPath}).github.user
-    : null;
-
   const answers = await inquirer.prompt([
     {
       type: 'input',
-      name: 'username',
-      message: 'What is your GitHub username?',
-      default: info.options['gh-username'] || githubUserFromGitConfig || npm.config.get('username'),
-      validate: _.ary(_.bind(validator.isLength, validator, _, 1), 1),
-    },
-    {
-      type: 'password',
-      name: 'password',
-      message: 'What is your GitHub password?',
-      validate: _.ary(_.bind(validator.isLength, validator, _, 1), 1),
+      name: 'token',
+      message: 'What is your GitHub personal access token?',
```

Comment:
Maybe change it to

```suggestion
      message: 'Provide a GItHub Personal Access Token (create a token at https://github.com/settings/tokens/new?scopes=repo)',
```

Reply:
I updated the message as requested @gr2m . I did some local testing to check that it could handle passed in `gh-token` as well as default to the answer supplied by the CLI prompt if needed. I didn't run the projejct end to end on a project of mine to test, should I do that? I wasn't sure how far to test.